default.hbs

```diff
@@ -76,23 +73,123 @@
         <script src="https://code.jquery.com/jquery-3.4.1.slim.min.js" integrity="sha384-J6qa4849blE2+poT4WnyKhv5vZF5SrPo0iEjwBvKU7imGFAV0wwj1yYfoRSJoZ+n" crossorigin="anonymous"></script>
         <script src="https://cdn.jsdelivr.net/npm/popper.js@1.16.0/dist/umd/popper.min.js" integrity="sha384-Q6E9RHvbIyZFJoft+2mJbHaEWldlvI9IOYy5n3zV9zzTtmI3UksdQRVvoxMfooAo" crossorigin="anonymous"></script>
         <script src="https://stackpath.bootstrapcdn.com/bootstrap/4.4.1/js/bootstrap.min.js" integrity="sha384-wfSDF2E50Y2D1uUdj0O3uMBJnjuUD4Ih7YwaYd1iqfktj0Uod8GCExl3Og8ifwB6" crossorigin="anonymous"></script>
-<<<<<<< HEAD
         <script src="https://cdn.jsdelivr.net/npm/bootstrap-select@1.13.14/dist/js/bootstrap-select.min.js"></script>
         <script src="{{asset "js/bootstrap-pagination.js"}}"></script>
-=======
         <script src="https://cdn.jsdelivr.net/gh/gmfmi/searchinghost-easy@latest/dist/searchinghost-easy-backpack.js"></script>
->>>>>>> upstream/master
 
         <script>
             // contentApiKey is instance-dependent; replace this with a new key if
             // testing on a local instance
             new SearchinGhostEasy({ contentApiKey: '3ecdb68993794bc0157e776e5d' });
 
+
+            // Filter feature
+            let primaryFilters = "";
+            let internalFilters = "";
+
+            function getFilterOptions(target){
+                // Retrieve select value from filter and manipulate the values for filtering
+                let filterOptions = $(target).val();
+                // Check if target is an internal tag filter
+                let internalTag = $(target).hasClass('internal-tag');
+                // Set respective prefix 
+                let tagPrefix = internalTag ? "tag-hash-" : "tag-";
+                // Initialise a new array for holding raw values to be written to localstorage
+                let rawFilter = new Array();
+                // Loop through each item in the filter
+                $.each(filterOptions, function(index,value) {
+                    // Push the raw value for localstorage
+                    rawFilter.push(value);
+                    // Basic slug generator: replace whitespaces with dashes, then strip punctuation
+                    filterOptions[index] = tagPrefix + value.replace(/\s+/g, '-').replace(/[.,'!?]/, '').toLowerCase();
+                })
+                if (internalTag){
+                    // Write values for internal tags
+                    internalFilters = filterOptions;
+                    // Write internal filters to localstorage for persistance across pages
+                    localStorage.setItem('internalStories', rawFilter.join("|"));
```

Comment:
Could `internalStories` be changed to `internalFilters`? Since it's the tags of the stories, and not the stories themselves, which have an internal/primary distinction.

Reply:
Fixed as part of 62bf77a2732fba5747e5434c4110844a120c146d